chacra/tests/async/test_rpm.py

```diff
@@ -0,0 +1,23 @@
+import mock
+from chacra.async import rpm
+
+
+class TestRPM(object):
+
+    @mock.patch('subprocess.check_call')
```

Comment:
there is no need to use mock, you can use pytest's `monkeypatch` fixture https://docs.pytest.org/en/latest/monkeypatch.html

Reply:
mock is in the py3.3 stdlib so I would prefer to use it (even if we have to install it currently from pypi) because it's the standard way of mocking things in python.
If you really want me to use `monkeypatch` I'll change it of course...

Reply:
Even if mock is in the standard library, this project is using Pytest, and uses the `monkeypatch` fixture everywhere. Please do not add a new dependency for patching and keep what the rest of the project is already using.

I don't think that using `mock` is the standard way of mocking things in Python.

Reply:
Done